### PR TITLE
Fixes Issue #257

### PR DESCRIPTION
**Pull request by goose000**
as described here https://github.com/docker/getting-started/issues/257#issuecomment-1079897370

### CODE REVIEW

**Review by farv — APPROVED** [review]
It worked perfectly for me. Thanks!

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+jinja2==3.0.3
 mkdocs==1.2.3
```
Comment (Mattinn) [inline requirements.txt:2]: No need to add jinja2 as a new dependency, mdocs patched it in release 1.2.4

**Comment by themhz** [comment]
I getting the same error and jinja2==3.0.3 didn't fix it for me
ERROR [build 2/2] RUN mkdocs build

**Comment by Mattinn** [comment]
> Hello! I am getting the same error and jinja2==3.0.3 didn't fix it for me ERROR [build 2/2] RUN mkdocs build
Instead of adding jinja2 as dependency > upgrade mdocs from 1.2.3 to 1.2.4 and rerun the build step

**Comment by StefanScherer** [comment]
Thank you for your help! 
After looking into this issue I've merged another PR that bumps mkdocs version which also fixes the issue.

**Pull request closed without merging**